namerd/docs/config.md

```diff
@@ -29,21 +22,32 @@ interfaces:
   port: 4321
 ```
 
-<a name="storage"></a>
-## Storage
+Welcome to the Configuration Reference for namerd!
 
-All configurations must define a **storage** key which must be a
-[storage](storage.md) object.
+namerd's configuration is controlled via config file, which must be provided
+as a command-line argument. It may be a local file path or `-` to
+indicate that the configuration should be read from the standard input.
+
+#### File Format
+
+The configuration may be specified as a JSON or YAML object.
 
-<a name="namers"></a>
-## Namers
+Key | Required | Description
+--- | -------- | -----------
+[admin](#administrative-interface) | no | Configures namerd's administrative interface. namerd admin has the same options as linkerd admin.
+[interfaces](#interfaces) | no | Configures namerd's published network interfaces.
+[storage](#storage) | yes | Configures namerd's storage backend.
+[namers](https://linkerd.io/config/latest/linkerd#namers) | no | Configures namerd's integration with various service discovery backends. namerd uses the same namers as linkerd.
```

Comment:
oops yeah I noticed that too. I'll hardcode it for now. I could hardcode it to `head` if we think that's a better alternative